libs/navigation/api/src/lib/navigation.service.spec.ts

```diff
@@ -0,0 +1,75 @@
+import {Test, TestingModule} from '@nestjs/testing'
+import {PrismaClient} from '@prisma/client'
+import {NavigationService} from './navigation.service'
+
+describe('NavigationService', () => {
+  let service: NavigationService
+  let prismaMock: any
+
+  beforeEach(async () => {
+    prismaMock = {
+      navigation: {
+        findUnique: jest.fn(),
+        findMany: jest.fn(),
+        create: jest.fn(),
+        delete: jest.fn(),
+        update: jest.fn()
+      },
+      navigationLink: {
+        deleteMany: jest.fn()
+      }
+    }
+
+    const module: TestingModule = await Test.createTestingModule({
+      providers: [NavigationService, {provide: PrismaClient, useValue: prismaMock}]
+    }).compile()
+
+    service = module.get<NavigationService>(NavigationService)
+  })
+
+  it('should get a navigation by id', async () => {
+    prismaMock.navigation.findUnique.mockResolvedValueOnce({})
+    await service.getNavigationById('123')
+    expect(prismaMock.navigation.findUnique).toHaveBeenCalled()
```

Comment:
Already tested by line below (same as the other tests)

Reply:
Its usually to narrow down if it's failed.

Reply:
But this will tell you it wasn't called or that it has been called, the snapshot tells you if it has been called and with what.